e2e-tests/gef/cypress/e2e/facility/about-facility.spec.js

```diff
@@ -217,13 +217,12 @@ context('About Facility Page', () => {
       cy.visit(relative(`/gef/application-details/${application.id}/facilities/${facilityId}/about-facility`));
       aboutFacility.facilityName().should('have.value', 'Name');
       aboutFacility.shouldCoverStartOnSubmissionNo().should('be.checked');
-      aboutFacility.coverStartDateDay().should('have.value', now.getDate());
-      aboutFacility.coverStartDateMonth().should('have.value', now.getMonth() + 1);
-      aboutFacility.coverStartDateYear().should('have.value', now.getFullYear());
-      aboutFacility.coverEndDateDay().should('have.value', now.getDate());
```

Comment:
I'd be super interested to know if there was an issue with exactly one year in the future (hopefully not!) or if this is done for simplicity of reading